components/VM_Arm/src/modules/init_ram.c

```diff
@@ -12,18 +12,12 @@
 #include <sel4vmmplatsupport/guest_memory_util.h>
 #include <vmlinux.h>
 
-extern unsigned long linux_ram_base;
-extern unsigned long linux_ram_size;
+extern unsigned long ram_base;
```

Comment:
Seem we could get this from a template header file then nicely?

Reply:
Depends on how we want to use the templates.

I could create wrapper functions that simply return a static variable contained in a template, or we can keep the `extern` method. Both work fine